src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -406,8 +405,12 @@ public virtual AuthenticationProperties ConfigureExternalAuthenticationPropertie
         /// <returns></returns>
         protected async virtual Task<bool> LogResultAsync(bool result, TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
         {
-            Logger.WriteInformation(Resources.FormatLoggingSigninResult(Resources.FormatLoggingResultMessage(methodName,
-                await UserManager.GetUserIdAsync(user)), result));
+            // Check if log level is enabled before creating the message.
```

Comment:
So I would say, the two 'success' results for SignIn manager are Success and TwoFactorRequired, all of the other results should be considered failures and logged like user manager failures.

I would try True/False logging in the same way as UserManager